microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -902,12 +912,54 @@ This is a programmatic way to record JavaScript execution, which can be useful i
 
 When DevTools is used to debug processes that are running in Node.js, this experiment makes whatever `console.profile` generates available to the **Performance** Tool.
 
+See also:
+* [JavaScript Profiler tool](../javascript-profiler/javascript-profiler-tool.md)
+
 Status:
-* This checkbox is present in Microsoft Edge Canary 122.
-* This checkbox is present in Microsoft Edge Stable 120.
+* This checkbox is not present in Microsoft Edge Canary 125.
+* This checkbox is present in Microsoft Edge Stable 123.
+
+
+<!-- ====================================================================== -->
+## Timeline: Enable track confirmation feature that can reorder or hide a track in the flame chart
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 125_
 
+Enables the track confirmation feature, which can reorder or hide a track in the flame chart.
+<!-- todo: need desc 
```

Comment:
Timeline: Enable track confirmation feature that can reorder or hide a track in the flame chart
Need real description, if possible.  What tool is this about?
@captainbrosset

Reply:
See comment above.